website/docs/transforms/mock.md

```diff
@@ -163,8 +163,27 @@ transforms:
           store:
```

Comment:
This is great!
How is the store populated? This method retrieves a user from the store, but how did the user get there in the first place?

Also, what would be the API to mock the array of users and provide custom data for them?